learn/tanssi/staking.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Staking
+description: Learn how Tanssi implements a novel Staking mechanism to provide a decentralized and trustless set of block producers to all Tanssi-powered Appchains
+---
+
+# Tanssi Staking {: #tanssi-staking }
+
+## Introduction {: #introduction }
+
+One of Tanssi's core propositions is to simplify the infrastructure complexity for Appchains. A significant component is bootstrapping a decentralized set of block producers, which Tanssi offers through its unique architecture and staking mechanics.
+
+Tanssi staking mechanics guarantee that the block producers for Tanssi and Tanssi-powered Appchains are selected through a trustless and decentralized mechanism. They also incentivize the community to delegate to top-performing or engaged block producers.
```

Comment:
I prefer the current wording because people delegate to either top-performing (in terms of rewards, I guess) or engaged block producers (like engaged community members)